jupyter-book/cellular_structure/annotation.yml

```diff
@@ -3,13 +3,18 @@ name: best_practices_annotation
 channels:
   - conda-forge
   - bioconda
+  - anaconda
 dependencies:
   - conda-forge::python=3.9
-  - conda-forge::scanpy=1.9
+  - conda-forge::scanpy>=1.9.3
   - conda-forge::leidenalg
   - conda-forge::jupyterlab
-  - conda-forge::jupyter_server=1.18.1
+  - conda-forge::jupyter_server>=1.21 # was: =1.18.1
```

Comment:
I don't think that we even need this dependency here

Reply:
it threw an error with my older version

Reply:
Yeah but why do we need to specify it at all?

Reply:
because if I don't specify it, it might install the wrong version and start throwing errors. That's what happened before I added the line to the yaml